README.md

```diff
@@ -146,7 +146,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 
 ### P
 
-* [PhantomJS](/glossary/PHANTOMJS.md): *to be completed.*
+* [PhantomJS](/glossary/PHANTOMJS.md): a headless WebKit scriptable with a JavaScript API.
```

Comment:
I find this sentence, while perfectly accurate, not very easy to understand. Do you think we could make it more within reach?